lib/templates/_footer.html

```diff
@@ -47,10 +47,16 @@
   }
 }
 
+function shiftWindow() {
```

Comment:
maybe leave a comment about what this is fixing and why this is here. Link to the bug?